src/lib/bootloader/grub2base.rb

```diff
@@ -64,14 +64,14 @@ def pmbr_setup(*devices)
       return if @pmbr_action == :nothing
 
       action_parted = case @pmbr_action
-      when :add    then "on"
-      when :remove then "off"
+      when :add    then true
+      when :remove then false
       else raise "invalid action #{action}"
       end
 
-      devices.each do |dev|
-        Yast::Execute.locally("parted", "-s", dev, "disk_set", "pmbr_boot", action_parted)
-      end
+      staging = Y2Storage::StorageManager.instance.y2storage_staging
+      disks = staging.disks.select { |d| devices.include?(d.name) }
+      disks.each { |d| d.pmbr_boot = action_parted }
```

Comment:
so, I need to do it set this flags in proposal and let storage do it during its write section and for runtime, call it manually. Right? Or can I call it always in this location, even in bootloader installation in first stage?